src/Composer.php

```diff
@@ -33,6 +33,10 @@ public function getDependencies($composerLockPath, $excludeDev = false)
             $packages = array_merge($json['packages'], isset($json['packages-dev']) ? $json['packages-dev'] : []);
         }
 
+        if (empty($packages)) {
+            return [];
```

Comment:
Pretty niche/unlikely scenario, but ran right into it with my test .lock file, lol
When there are only dev packages, and the `--no-dev` flag is true, without this change it throws:
```
[ERROR] Vulnerabilities scan failed with exception: array_merge() expects at least 1 parameter, 0 given